packages/eslint-config/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@kilohealth/eslint-config",
+  "version": "1.0.1",
+  "description": "Eslint preset",
+  "keywords": [
+    "react-native",
+    "eslint",
+    "eslintconfig",
+    "prettier",
+    "react",
+    "node"
+  ],
+  "homepage": "https://github.com/kilohealth/eslint-config",
+  "repository": "https://github.com/kilohealth/eslint-config",
+  "license": "MIT",
+  "author": "Kilo.Health",
+  "main": "./react-native.js",
```

Comment:
Node config is a base for React & RN, so my suggestion would be to change the main source to `./node.js`